postgres/init.sql

```diff
@@ -1,8 +1,25 @@
-CREATE DATABASE metricity;
-
-\c metricity;
-
-CREATE TABLE users (
+DO
+$do$
+DECLARE
+  _db TEXT := %s;
+  _user TEXT := %s;
+  _password TEXT := %s;
+BEGIN
+  CREATE EXTENSION IF NOT EXISTS dblink;
+  IF EXISTS (SELECT 1 FROM pg_database WHERE datname = _db) THEN
+    RAISE NOTICE 'Database already exists';
+  ELSE
+    PERFORM dblink_connect(
+        'host=localhost user=' || _user ||
+        ' password=' || _password ||
+        ' dbname=' || current_database()
+    );
+    PERFORM dblink_exec('CREATE DATABASE ' || _db);
+  END IF;
+END
+$do$;
```

Comment:
If you got the code from SO, then you should [attribute it](https://stackoverflow.blog/2009/06/25/attribution-required/) to the author(s).

Reply:
Well maybe [it's complicated](https://meta.stackexchange.com/questions/272956/a-new-code-license-the-mit-this-time-with-attribution-required?cb=1)...

Reply:
Hmm yea that a little contradicting.
Maybe just the below at the top of the sql file will do it?
```
-- The following function is from Stack Overflow
-- https://stackoverflow.com/questions/18389124/simulate-create-database-if-not-exists-for-postgresql/36218838#36218838
-- User frnakhommers (https://stackoverflow.com/users/971229/frankhommers)
```

Reply:
Yes, probably. I wouldn't worry about it too much as long as we at least tried to attribute :)

Reply:
Added in https://github.com/python-discord/site/pull/583/commits/5dfd9e640ca11207a4e9d4178dab3fad960b4bd4

Reply:
Miss-spelled the user's name, force pushed https://github.com/python-discord/site/pull/583/commits/04ab6e6e10eace3ee103cef61f2ee94f3d2129fb lol